src/dockerfiles/lambda/connectors/athena.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM public.ecr.aws/lambda/python:3.8
+
+# Copy function code
+COPY lambda/connectors/data.py ${LAMBDA_TASK_ROOT}
+
+# Install the function's dependencies using file requirements.txt
+# from your project folder.
+
+COPY lambda/requirements.txt  .
+RUN pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
+RUN pip3 install awswrangler --target "${LAMBDA_TASK_ROOT}"
+
+RUN pip3 install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ aqueduct-ml-test-hari==0.0.1 --target "${LAMBDA_TASK_ROOT}"
```

Comment:
Change this to aqueduct-ml once the latest changes in this PR are cut in a release.

Reply:
Will we hard-code release version here? This doc may need to update: https://www.notion.so/aqueducthq/Runbook-Cutting-an-OSS-Release-b72400b7584845f0b0b4ff84bab4aedc (@cw75 )

Reply:
Yes we should. @hsubbaraj-spiral could you hardcode the version number just like what we are doing for the k8s images and update the runbook to contain steps to build and push images? Thx!

Reply:
Yep, I'll change this before merging to main, just want to be able to test the whole branch together before.